src/mscorlib/src/System/Char.cs

```diff
@@ -927,12 +927,17 @@ public static String ConvertFromUtf32(int utf32)
                 // This is a BMP character.
                 return (Char.ToString((char)utf32));
             }
-            // This is a sumplementary character.  Convert it to a surrogate pair in UTF-16.
-            utf32 -= UNICODE_PLANE01_START;
-            char[] surrogate = new char[2];
-            surrogate[0] = (char)((utf32 / 0x400) + (int)CharUnicodeInfo.HIGH_SURROGATE_START);
-            surrogate[1] = (char)((utf32 % 0x400) + (int)CharUnicodeInfo.LOW_SURROGATE_START);
-            return (new String(surrogate));
+
```

Comment:
Also, the method needs to be marked with `SecuritySafeCritical` critical attribute now because of it contains unsafe code.